Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -80,13 +89,38 @@ public final class ProjectPamphletViewModel: ProjectPamphletViewModelType, Proje
     let freshProjectAndRefTag = freshProjectAndRefTagEvent.values()
 
     let ctaButtonTapped = freshProjectAndRefTag
-      .takeWhen(self.backThisProjectTappedProperty.signal)
-      .map { project, refTag in
-        (project, refTag)
+      .takePairWhen(self.pledgeCTAButtonTappedProperty.signal)
+      .map(unpack)
+
+    let goToManagePledge = ctaButtonTapped
+      .filter { canShowManageViewPledgeScreen($0.0, state:$0.2) }
```

Comment:
Looking at this in action, it might actually be more readable to name those params. I'll leave this out to you, I'm OK with both - but wanted to raise it in case you were in favour of the other option.

`.filter { project, _, state in canShowManageViewPledgeScreen(project, state: state) }`

(this of course would apply to other places in this PR.